patterns/1-initial/code-of-conduct.md

```diff
@@ -0,0 +1,77 @@
+## Title
```

Comment:
Answering the `Is there a general description of the goals of a CoC that we can link to?` question, this [reference](https://opensource.guide/code-of-conduct/) is a good one from the Open Source space, but I don't see how to link here. Maybe adding this reference in the Related patterns...

Regarding to broken the CoC, the implementation section is extended including the reporting process and the consequences when the CoC is not followed. It is a general description, just because, each CoC can differ in the final implementation of these processes, however, all of them must be part of the CoC.

Reply:
I think it doesn't hurt to add it under References at the end of this pattern.

Actually i renamed "Related Patterns" to "References" and added the link in. Please take a look if you like the approach.